addons/website/static/src/components/dialog/seo.js

```diff
@@ -267,6 +267,10 @@ class TitleDescription extends Component {
 
         this.maxRecommendedDescriptionSize = 300;
         this.minRecommendedDescriptionSize = 50;
+
+        useEffect(() => {
+            document.title = this.title;
+        }, () => [this.seoContext.title]);
```

Comment:
Cool... but then I hit the cancel button of the SEO dialog and the entered title is not discarded... :grimacing: